scripts/node-entrypoint.sh

```diff
@@ -27,5 +29,22 @@ psql -v ON_ERROR_STOP=1 \
 CREATE EXTENSION IF NOT EXISTS btree_gist;
 EOF
 
-# run the main node
-env node /vendor/subql-cosmos/packages/node/bin/run "$@"
\ No newline at end of file
+if [ "$WATCH" = "true" ]
+then
+  # call the first command if WATCH is true
+  echo "WATCH is true. Running the application with nodemon..."
```

Comment:
What do you think about adding another log line if `NODE_ENV` is "production" that says something like `WARN: Hot-reload is not recommended in production. Unset $WATCH to disable it`?

Reply:
1000% true, let me add it.